lib/intelligent_foods/resources/order.rb

```diff
@@ -9,8 +9,12 @@ class Order < IntelligentFoods::Object
     INITIALIZED = "initialized"
     PROCESSED = "processed"
 
-    def initialize(args = {})
+    attr_reader :skip_temperature_check, :skip_address_check
```

Comment:
i won't nitpick it, but is there any reason not to have used attr_accessor here?